app/steps/ui/tasklist/includes/copies.html

```diff
@@ -1,7 +1,7 @@
 {% set previousTaskStatus = fields.previousTaskStatus.value.CopiesTask %}
 
 <li class="task-item">
-    <span class="heading-medium">{{content.copiesTask}}</span>
+    <h3 class="heading-medium">{{content.copiesTask}}</h3>
```

Comment:
It would be more semantically correct for these headings to be `h2` as in your previous commit.